app/Database/Schema/UserSchema.php

```diff
@@ -3,10 +3,12 @@
 
 namespace App\Database\Schema;
 
+use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\ORM\Mapping as ORM;
 
 /**
  * @method bool isActive()
+ * @method null|\App\Database\Schema\Account[] getAccounts()
```

Comment:
Please match annotation to what the type actually is. Here `account` is instantiated as `ArrayCollection` so the annotation should be ArrayCollection's interface which is `\Doctrine\Common\Collections\Collection`